static/app/components/group/tagFacets/tagFacetsDistributions.tsx

```diff
@@ -65,7 +64,10 @@ export default function TagFacetsDistributions({
   }, [api, JSON.stringify(environments), groupId, tagKeys]);
 
   const tagsData = tagFormatter?.(state.tagsData) ?? state.tagsData;
-  const sortedTagKeys = Object.keys(tagsData).sort();
+  const topTagKeys = tagKeys.filter(tagKey => Object.keys(tagsData).includes(tagKey));
+  const allTagKeys = Object.keys(tagsData)
```

Comment:
```suggestion
  const remainingTagKeys = Object.keys(tagsData)
```

Small thing, I felt `allTagKeys` could've been confusing down the line